addons/mail/models/mail_thread.py

```diff
@@ -4031,11 +4040,21 @@ def _notify_get_recipients(self, message, msg_vals=False, **kwargs):
             if notify_author_mention and skip_author_id in pids:
                 skip_author_id = False
 
+        # avoid double email notification if already emailed in original email
+        emailed_normalized = [
+            email_normalize(addr) for addr in email_split(
```

Comment:
`email_normalize_all` do already `email_normalize` + `email_split`, can we use it ?
(we will still need to merge `incoming_email_to` and `incoming_email_cc` though)

Reply:
Good idea :) too much normalize stuff those weeks, I am making noeuds sometimes.